packages/neon_lints/lint_maker.yaml

```diff
@@ -0,0 +1,29 @@
+dart:
+  commitHash: 7f40f11
```

Comment:
I'm wondering if there is any way to extract this information from the Dart SDK that currently executes the script. That would be pretty neat (as it would also avoid rules that don't work yet) but it's not very realistic that this is possible in any way :thinking:

Reply:
I mean the analyzer must have the ability the question is whether it exposes it

Reply:
The ability to know from which commit hash it was built? I'm not sure about that

Reply:
Not that but what rules are available in the used release.

I mean it has to know it otherwise it couldn't warn us of of the unreleased rules.